WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsActivity.kt

```diff
@@ -98,7 +98,7 @@ class NotificationsSettingsActivity : LocaleAwareActivity(), MainSwitchToolbarLi
         toolbar?.let { setSupportActionBar(it) }
 
         supportActionBar?.apply {
-            setTitle(string.notification_settings)
+            setTitle("")
```

Comment:
We can remove the setTitle line instead of setting a blank title. Same for the setTitle line on the `ReaderSubsActivity`.

Reply:
HI Irfan if we remove the setTitle() then it will show the Wordpress in titlebar that's why we have to give default empty string

Reply:
🤔 Good point! The reason for that behavior is `setDisplayHomeAsUpEnabled(true)`. I set it to `false` and tested it, it worked as I expected. Can you test it too? If it passes on your test too, you can update `ReaderSubsActivity` too.

https://github.com/wordpress-mobile/WordPress-Android/blob/7e7b3ddc16400ea7f059ec416d0574a4ead360b9/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsActivity.kt#L102

Reply:
> > HI Irfan if we remove the setTitle() then it will show the Wordpress in titlebar that's why we have to give default empty string
> 
> 🤔 Good point! The reason for that behavior is `setDisplayHomeAsUpEnabled(true)`. I set it to `false` and tested it, it worked as I expected. Can you test it too? If it passes on your test too, you can update `ReaderSubsActivity` too.
> 
> https://github.com/wordpress-mobile/WordPress-Android/blob/7e7b3ddc16400ea7f059ec416d0574a4ead360b9/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsActivity.kt#L102

if we make it false then back arrow navigation will be removed from screen.

Reply:
> > HI Irfan if we remove the setTitle() then it will show the Wordpress in titlebar that's why we have to give default empty string
> 
> 🤔 Good point! The reason for that behavior is `setDisplayHomeAsUpEnabled(true)`. I set it to `false` and tested it, it worked as I expected. Can you test it too? If it passes on your test too, you can update `ReaderSubsActivity` too.
> 
> https://github.com/wordpress-mobile/WordPress-Android/blob/7e7b3ddc16400ea7f059ec416d0574a4ead360b9/WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsActivity.kt#L102

instead of setDisplayHomeAsUpEnabled I tried making false to setDisplayShowTitleEnabled and it worked as expected now.

Reply:
> if we make it false then back arrow navigation will be removed from screen.
> instead of setDisplayHomeAsUpEnabled I tried making false to setDisplayShowTitleEnabled and it worked as expected now.

I would say `setDisplayShowTitleEnabled` instead of `setDisplayHomeAsUpEnabled` 🤦🏻‍♂️
Thank you for correcting my misleading comment. 🙂